bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/Panel.java

```diff
@@ -36,4 +36,15 @@ public interface Panel extends Container, ContainerConstraint {
     default Boolean isReadOnly() {
         return null;
     }
+
+    /**
+     * Checks if the container is wrap data.
+     *
+     * @return {@code true} if the container is wrap data, {@code false} otherwise
+     * @since com.adobe.cq.forms.core.components.models.form 5.3.0
+     */
+    default boolean isWrapData() {
```

Comment:
We need to consume this on server, so its not required to expose this in json

Reply:
I will add jsonIgnore

Reply:
Done